app/styles/ember-frost-bunsen/frost-date-picker.scss

```diff
@@ -0,0 +1,8 @@
+.frost-bunsen-has-value {
```

Comment:
Yeah don't add it here. If the default looks like garbage the correct thing todo is add the improved styling to the `ember-frost-date-picker` addon itself. Otherwise we'll have inconsistencies between apps using/not using bunsen.